packages/config/package.json

```diff
@@ -17,30 +17,35 @@
     },
     "homepage": "https://github.com/RedHatInsights/frontend-components/tree/master/packages/config#readme",
     "dependencies": {
-        "babel-loader": "^8.0.6",
-        "clean-webpack-plugin": "^2.0.2",
-        "connect-history-api-fallback": "^1.6.0",
-        "css-loader": "^2.1.1",
-        "eslint-loader": "^2.2.1",
-        "file-loader": "^3.0.1",
-        "git-revision-webpack-plugin": "^3.0.3",
-        "html-replace-webpack-plugin": "^2.5.5",
-        "html-webpack-plugin": "^3.2.0",
-        "koa-connect": "^2.0.1",
-        "lodash-webpack-plugin": "^0.11.5",
-        "mini-css-extract-plugin": "^0.6.0",
-        "node-sass": "^4.12.0",
+        "assert": "^2.0.0",
+        "babel-loader": "^8.2.1",
+        "browserify-zlib": "^0.2.0",
+        "buffer": "^6.0.2",
+        "clean-webpack-plugin": "^3.0.0",
+        "css-loader": "^5.0.1",
+        "eslint-webpack-plugin": "^2.3.0",
+        "file-loader": "^6.2.0",
+        "git-revision-webpack-plugin": "^3.0.6",
+        "html-replace-webpack-plugin": "^2.6.0",
+        "html-webpack-plugin": "^4.5.0",
+        "koa-connect": "^2.1.0",
+        "mini-css-extract-plugin": "^1.3.1",
+        "node-sass": "^5.0.0",
+        "path-browserify": "^1.0.1",
+        "process": "^0.11.10",
         "react-router-sitemap": "^1.2.0",
-        "sass-loader": "^7.1.0",
-        "source-map-loader": "^0.2.4",
-        "style-loader": "^0.23.1",
-        "ts-loader": "^6.2.1",
-        "webpack-dev-server": "^3.4.1",
-        "webpack-hot-client": "^4.1.1",
-        "webpack-serve": "1.0.2",
-        "write-file-webpack-plugin": "^4.5.0"
+        "sass-loader": "^10.1.0",
+        "source-map-loader": "^1.1.2",
+        "stream-browserify": "^3.0.0",
+        "style-loader": "^2.0.0",
+        "ts-loader": "^8.0.11",
+        "util": "^0.12.3",
+        "webpack": "^5.6.0",
```

Comment:
@karelhala we have to bump the webpack version to latest due to the issue with dev-server